README.md

```diff
@@ -1,22 +1,50 @@
-# Nuxt Minimal Starter
-
-Look at the [Nuxt documentation](https://nuxt.com/docs/getting-started/introduction) to learn more.
+# PhyExDB App
 
 ## Setup
 
-Make sure to install dependencies:
+Make sure to install dependencies, ideally use `npm ci` to install the exact versions specified in `package-lock.json`:
 
 ```bash
-npm install
+npm ci
 ```
+
+Only use `npm install` if you want to update the dependencies or add new ones.
+
+### Environment Variables
+
+Create a `.env`, `.env.test` and `.env.development` file in the root directory of the project. Fill them with the values of the `.env.example` file.
+
+In the `.env` file change `localhost` in the `DATABASE_URL` to `db` for the docker container to work correctly.
```

Comment:
I think this line comes from a time when we used sequelize

Reply:
Line 17

Reply:
No this is up to date, in the .env we have this variable `DATABASE_URL=postgresql://nuxt:nuxt_password@localhost:5432/phyexdb` and in it localhost should be changed to db in the .env file, for the other envs localhost is correct